graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/dml/DataTypeMapping.java

```diff
@@ -129,6 +130,39 @@ private static void formatUdt(
     out.append('}');
   }
 
+  private static void formatTuple(
+      Column.ColumnType type, Object value, NameMapping nameMapping, StringBuilder out) {
+    out.append('(');
+    @SuppressWarnings("unchecked")
+    Map<String, Object> mapValue = (Map<String, Object>) value;
+    List<Column.ColumnType> subTypes = type.parameters();
+
+    // Track null values.
+    boolean hasANullItem = false;
+
+    for (int i = 0; i < subTypes.size(); i++) {
+      Object item = mapValue.get("item" + i);
+
+      if (item == null) {
+        hasANullItem = true;
+        continue;
+      }
+
+      if (hasANullItem) {
+        throw new UnsupportedOperationException(
+            "Tuple can't have a null item followed by a non-null item");
```

Comment:
Why the restriction? I tried in cql and it works fine.

Reply:
You're right, we don't need that restriction.
It's my fault, I left a completely misleading comment on the original issue: https://github.com/stargate/stargate/issues/233#issuecomment-722698412

Reply:
Oh, my bad, I'll fix it.